wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -14,7 +14,9 @@
 
 {% block content %}
     {% trans "Styleguide" as title_trans %}
-    {% include "wagtailadmin/shared/header.html" with title=title_trans %}
+    {% with icon='image' %}
+      {% include "wagtailadmin/shared/header.html" with title=title_trans %}
```

Comment:
`{% include "wagtailadmin/shared/header.html" with title=title_trans icon='image' %}` ?

Reply:
Yes, this is in the website frontend, not in the Wagtail admin. So the icon symbol needs to be added.

See the source of Wagtail admin page too. It contains all icons.

Reply:
Sorry, should have been explicit.. I meant it you could fold the `{% with %}` into the include. Not a biggie

Reply:
Yes! Done.